wp-content/themes/twentyseventeen-wp20/style.css

```
@@ -410,20 +413,42 @@ body.page .entry-header {
 	}
 }
 
+/* Override base styles */
+.navigation-top .wrap {
+	box-sizing: content-box;
+	padding: 0;
+}
+
+@media screen and (min-width: 48em) {
+	.navigation-top .wrap {
+		padding-left: var(--wp20--spacing--edge-space);
+		padding-right: var(--wp20--spacing--edge-space);
+	}
+}
```

Given that the horizontal padding is the default behaviour I think we could simplify this by only setting the padding to 0 on small devices:
```suggestion
}

@media screen and (max-width: 48em) {
	.navigation-top .wrap {
		padding: 0;
	}
}
```

---

Since there are still a couple of paddings applied from twentyseventeen that would mess up the look here, the styles here are still needed.